pkg/cephfs/nodeserver.go

```diff
@@ -177,6 +179,31 @@ func (ns *NodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	volCap := req.GetVolumeCapability()
```

Comment:
this check is already done here https://github.com/ceph/ceph-csi/blob/master/pkg/cephfs/util.go#L168

Reply:
will remove that. Thanks.

Reply:
@Madhu-1 updated the PR.